packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-custom-pwd-recovery-mfa/main/nodeexpress/siw.md

```diff
@@ -88,7 +88,7 @@ After set up, render the client site page and pass the configurations to the wid
 
 After the widget is loaded, the following reset page is appears:
 
-<div class="common-image-format bordered-image">
```

Comment:
CSS border doesn't match the rounded border radius of the image.

Reply:
intrinsic border removed

Reply:
fixed